cypress/integration/explore.spec.js

```diff
@@ -1,3 +1,6 @@
+import { SearchParameters } from '../../src/types/search.types';
+import { ResourceFeatureTypes } from '../../src/types/resource.types';
+
```

Comment:
vi kunne også lagt til 
  beforeEach(() => {
    cy.visit('/');
  });

Reply:
Er vel to av testene man ikke starter på "/"

Reply:
vi kan jo fortsatt ha dem - så har vi beforeeach som en failsafe om det blir avglemt i noen metoder ?

Reply:
Fikset